docs/testing/webdriverio/01-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WebdriverIO Overview
+sidebar_label: Overview
+---
+
+# Overview
+
+WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and stable test suite.
+
+Testing with WebdriverIO has the following advantages:
+
+- __Cross Browser Support__: WebdriverIO is designed to support all platforms, either on desktop or mobile. You can run tests on actual browser your users are using, including covering different versions of them.
+- __Real User Interaction__: Interacting with elements in WebdriverIO through the WebDriver protocol is the closes you can get to reality compared to emulated interaction in virtual DOM environments.
+- __Web Platform Support__: Running tests in actual browser allows you to tap into the latest Web Platform features for testing your components, often not available when using virtual DOM environments.
+
+## Set Up
+
+To get started with WebdriverIO, all you need to do is to run their project starter:
+
+```sh
+npm init wdio@latest .
```

Comment:
this is fun haha

<img width="671" alt="Screenshot 2024-02-07 at 2 17 46 PM" src="https://github.com/ionic-team/stencil-site/assets/6207644/2068b2b8-c740-46f6-b755-fd4692cd4ee4">